src/main/java/com/commercetools/sync/inventories/InventoryService.java

```diff
@@ -0,0 +1,69 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletionStage;
+
+/**
```

Comment:
throughout whole PR you often provide comments with very little or no meaningful information about the class.
here it's clear what interface hides by just looking at its name. Maybe you could just leave it with comment like:
Provides CRUD operations upon {@link InventoryEntry}.